CHANGELOG.md

```diff
@@ -1,3 +1,139 @@
+* [FEATURE] Allow configuring TLS options in scrape configurations.
+* [FEATURE] Add instrumentation around configuration reloads.
+* [FEATURE] Add `bool` modifier to comparison operators to enable boolean
+  (`0`/`1`) output instead of filtering.
+* [FEATURE] In Zookeeper serverset discovery, provide `__meta_serverset_shard`
+  label with the serverset shard number.
+* [FEATURE] Provide `__meta_consul_service_id` meta label in Consul service
+  discovery.
+* [FEATURE] Allow scalar expressions in recording rules to enable use cases
+  such as building constant metrics.
+* [FEATURE] Add `label_replace()` query language function.
+* [FEATURE] In Consul service discovery, fill in the `__meta_consul_dc`
+  datacenter label from the Consul agent when it's not set in the Consul SD
+  config.
+* [FEATURE] Scrape all services upon empty services list in Consul service
+  discovery.
+* [FEATURE] Add `labelmap` relabeling action to map a set of input labels to a
+  set of output labels using regular expressions.
+* [FEATURE] Introduce `__tmp` as a relabeling label prefix that is guaranteed
+  to not be used by Prometheus internally.
+* [FEATURE] Kubernetes-based service discovery.
+* [FEATURE] Marathon-based service discovery.
+* [FEATURE] Support multiple series names in console graphs JavaScript library.
+* [FEATURE] Allow reloading configuration via web handler at `/-/reload`.
+* [FEATURE] Updates to promtool to reflect new Prometheus configuration
+  features.
+* [FEATURE] Add `proxy_url` parameter to scrape configurations to enable use of
+  proxy servers.
+* [FEATURE] Add console templates for Prometheus itself.
+* [FEATURE] Allow relabeling the protocol scheme of targets.
+* [FEATURE] Add `predict_linear()` query language function.
+* [FEATURE] Support for authentication using bearer tokens, client certs, and
+  CA certs.
+* [FEATURE] Implement unary expressions for vector types (`-foo`, `+foo`).
+* [FEATURE] Add console templates for the SNMP exporter.
+* [FEATURE] Make it possible to relabel target scrape query parameters.
+* [FEATURE] Add support for `A` records in DNS service discovery.
+* [ENHANCEMENT] Update vendored dependencies.
+* [ENHANCEMENT] Add compression to more HTTP handlers.
+* [ENHANCEMENT] Make -web.external-url flag help string more verbose.
+* [ENHANCEMENT] Improve metrics around remote storage queues.
+* [ENHANCEMENT] Use Go 1.5.1 instead of Go 1.4.2 in builds.
+* [ENHANCEMENT] Update the architecture diagram in the `README.md`.
+* [ENHANCEMENT] Time out sample appends in retrieval layer if the storage is
+  backlogging.
+* [ENHANCEMENT] Make `hash_mod` relabeling action use MD5 instead of FNV to
+  enable better hash distribution.
+* [ENHANCEMENT] Better tracking of targets between same service discovery
+  mechanisms in one scrape configuration.
+* [ENHANCEMENT] Handle parser and query evaluation runtime panics more
+  gracefully.
+* [ENHANCEMENT] Add IDs to H2 tags on status page to allow anchored linking.
```

Comment:
I'll clarify that this is actually a user-requested feature (to be able to link to them).